README.md

```
@@ -31,13 +31,13 @@ Before installing the web app, you need to set up the Fineract backend server:
 
 1. **Choose ONE of these backend options:**
    - **Option A: Use existing remote server**
-     - Use the sandbox (MariaDB) at https://sandbox.mifos.community ** System is restored every 6 hours **
-     - Use the demo (MariaDB) at https://demo.mifos.community
-     - Use the demo (Keycloak) at https://oauth.mifos.community
-     - Use the demo (2FA) at https://2fa.mifos.community
-     - Use the demo (Oidc) at https://oidc.mifos.community
-     - Use the demo (Postgres) at https://elephant.mifos.community
-     - Configure to your server by updating API URLs in environment files
+   - Use the sandbox (MariaDB) at https://sandbox.mifos.community — sandbox data is reset every 6 hours; test data and transient state may be cleared.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)